### PR TITLE
Balance Component in new Hero Balance View

### PR DESCRIPTION
**Pull request by lhr0909**
I have broken out the balance parts into a separate component.

Had some small tweaks on the css class names, so we can have more generic names for the component, easier to style it in different views.

desktop view

image

mobile view (account name and image is wired too)

image

### CODE REVIEW

**Comment by lhr0909** [comment]
adding tests. The current component setup is hard to write unit tests, so I am going to refactor a small bit.

**Comment by lhr0909** [comment]
@sdtsui should be ready for review on this part

File: ui/app/components/balance-component.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+BalanceComponent.prototype.render = function () {
+  const props = this.props
+  const { balanceValue } = props
+  const needsParse = 'needsParse' in props ? props.needsParse : true
+  const formattedBalance = balanceValue ? formatBalance(balanceValue, 6, needsParse) : '...'
+
+  return h('div.balance-container', {}, [
+    // laptop: 50px 50px
```
Comment (sdtsui) [inline ui/app/components/balance-component.js:21]: styling is complete, feel free to make a commit to latest (no need for PR/review process), so unnecessary comments can be removed.

we can also clean it up later, no biggie either way

File: test/unit/components/balance-component-test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+var assert = require('assert')
+var BalanceComponent = require('../../../ui/app/components/balance-component')
+
+describe('BalanceComponent', function () {
```
Comment (sdtsui) [inline test/unit/components/balance-component-test.js:4]: See Slack for a note on testing.

**Review by sdtsui — APPROVED** [review]
:+1:
Taking some minor comments offline since we already got channels for that, merging now. Thanks.

**Pull request merged**